src/veh_interact.h

```diff
@@ -172,6 +172,9 @@ class veh_interact
         void allocate_windows();
         void do_main_loop();
         void deallocate_windows();
+
+        /*Filter stuff*/
+        std::string filter;
```

Comment:
You want me to get rid of the filter in the interface?

I'm really not sure, that's part of the state, isn't it?